views/models/UserModel/index.js

```diff
@@ -0,0 +1,30 @@
+import { BASE_REQUEST_OPTIONS } from '../../../constants'
+import { apiRoutes } from '../../routes/routes'
+
+const UserModel = {
+  show: async (userId) => {
+    const response = await fetch(apiRoutes.users.user(userId), {
+      ...BASE_REQUEST_OPTIONS,
+    })
+    if (response.status !== 200) throw new Error(response.json().error)
```

Comment:
we're calling response.json twice in this function. Also, that returns a promise so we need to await it.